tests/Types/AsciiTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Types;
+
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\AsciiType;
+use PHPUnit\Framework\TestCase;
+
+class AsciiTest extends TestCase
```

Comment:
Please name `AsciiStringTypeTest` according to the class being tested.

Reply:
@morozov should be name this new type `AsciiType` or `AsciiStringType`? I tend to prefer the former but let me know either way.

Reply:
I'd prefer `AsciiStringType` and `'ascii_string'`:
1. They extend the String type.
2. Technically, there may be other ASCII-based types (e.g. ASCII Text).